README.md

```diff
@@ -1,205 +1,278 @@
+  * Coverage checks look to see that the percentage of line of code covered by unit tests has _increased_. If it hasn't the test will not pass.
+  * Platform build checks are tested on a Unix and Windows Servers using Travis and Appveyor, respectively. Travis and Appveyor are servers with these OSs where the program is built and checked. This is useful because if, for example, SOILWAT2 is not building on an individual's Linux computer, but is building on Travis, we know the user's computer is not configured correctly, as opposed to a software design issue.
+
+* __Integration Tests:__ Integration tests, test at the comprehensive level to check that new code or feature is still producing the same or sensible output. Some integrChecks are made on speed and against reference databases.
+
+### R Code Standards <a name="rinfo"/>
+
+#### Style Guide <a name="rstyle"/>
+
+* __Style guide__: We aim to follow [Hadley Wickham's style guide for R](http://adv-r.had.co.nz/Style.html). This is a work in progress.
+
+To enforce certain elements of this style guide are R repositories now use the [lintr package.](https://github.com/jimhester/lintr) Specifically, we are enforcing *** *** with lintR (these can be found in the .lintr file). Tutorials on how to understand lintr markers or warnings in RStudio and Atom is available in the lintr README.
+
+#### Code Development <a name="rdevel"/>
+
+Code development in our R repositories is complex. To navigate this complexity we use a combination of software tools ([RStudio](https://www.rstudio.com/products/rstudio/download/) and [Atom]()) and testing. This complexity is rooted in our software architecture where many times changes to R code has an effect on another R package and/or the C code. Our R packages revolve around increasing user functionality of our SOILWAT2 C model, so even if code changes only occur in R, you must be conscientious of potential downstream effects.
+
+For example, changes in rSOILWAT2 will almost always require changes to SOILWAT2 (i.e. so that SOILWAT2 can receive a new variable that is being passed from rSOILWAT2) and rSFSW2 (i.e. rSFSW2 would format site-specific information and then use a rSOILWAT2 function to format this information for passage to C).
+
+Here are some guidelines to assist in R code development:
+
+* If your code is a feature, open up a new project on GitHub and sketch out a plan (specific functions) for how you will implement new functionality. Ask for review from your supervisor.
+* Take advantage of [RStudio's debugger](https://support.rstudio.com/hc/en-us/articles/205612627-Debugging-with-RStudio) to walk through functions with similar functionality.
+  - Note: Ironically, sometimes the debugger can be buggy. With patience, you can still walk through functions.
+* Test your new code within an integration project.  
+  - Copy the test project from within the _/Git/_ folder to your _/Desktop_ so that changes to input files aren't integrated onto Git.
```

Comment:
Does the `test project` refer specifically to `rSFSW2`s `TestPrj4/`? What about developing R code for `rSOILWAT2`?